### PR TITLE
Launch docs

### PR DESCRIPTION
**Pull request by ngrayluna**
## Description
What does the pull request do? If it fixes a bug or resolves a feature request, be sure to link to that issue.
WIP Lauch docs

## Ticket
Does this PR fix an existing issue? If yes, provide a link to the ticket here:

## Checklist
Check if your PR fulfills the following requirements. Put an `X` in the boxes that apply. 

- [ ] Files I edited were previewed on my local development server with `yarn start`. My changes did not break the local preview.
- [ ] Build (`yarn build`) was run locally and successfully without errors or warnings.
- [ ] I merged the latest changes from `main` into my feature branch before submitting this PR.

### CODE REVIEW

File: docs/guides/launch/intro.md [hunk 1]
```diff
@@ -5,25 +5,33 @@ description: Easily queue and manage jobs using W&B Launch.
...
+:::caution
+W&B Launch is Public preview in active development. 
 
-:::danger
-Beta product in active development
+Talk to our Sales Team to get Launch up and running for your company: [https://wandb.ai/site/pricing](https://wandb.ai/site/pricing)
+:::
 
-Interested in Launch? Reach out to your account team to talk about joining the customer pilot program for W&B Launch.
+Use W&B Launch to start jobs in the compute  environment or infrastructure of choice from one central interface. W&B Launch removes the complexity of using different environments to run training, inference, and more. Weights and biases will automatically capture the information needed to reproduce the run, containerize the job and launch the job locally or remotely.
```
Comment (KyleGoyette) [inline docs/guides/launch/intro.md]: Should "biases" in this paragraph be capitalized?

File: docs/guides/launch/intro.md [hunk 2]
```diff
@@ -5,25 +5,33 @@ description: Easily queue and manage jobs using W&B Launch.
...
-:::
 
-Connect your own SageMaker or Kubernetes cluster, then easily queue and manage jobs using W&B Launch. Kick off jobs on your own infrastructure from the W&B UI or CLI. Applications to the Launch pilot program are open to paying customers at the moment. To get started with W&B for your company, talk to Sales here: [https://wandb.ai/site/pricing](https://wandb.ai/site/pricing)
+## How it works
+There are three unique concepts to W&B Launch: *jobs*, *queues*, and *agents*: 
+* **Job**:  a definition of a computational process. You can think of W&B Jobs as a ‘Run template’. A Job can have one or more versions. Each job version combines source code and environment to produce a template for a reproducible run.
+* **Queue**: FIFO queue of W&B Jobs. A launch queue has a compute resource associated to that queue. Launch queues are entity-scoped. All jobs pushed to a queue automatically posses the same compute resource type.
+* **Agents**: A launch agent listens the queue and executes jobs added to the queue. If certain criteria are met, the launch agent builds an image for the environment and sends the job to the desired compute runner.
```
Comment (KyleGoyette) [inline docs/guides/launch/intro.md]: We could change "if certain criteria are met" to "if you don't already have an image, we'll build one." or something like that.

File: docs/guides/launch/prerequisites.md [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+A launch agent listens the queue and executes jobs added to the queue. If certain criteria are met, the launch agent builds an image for the environment and sends the job to the desired compute runner. 
+
+The following tabs provide information on how to activate a launch agent with basic, standard control settings and with more control options with a config file.
+
+
+<Tabs
+  defaultValue="basic"
+  values={[
+    {label: 'Basic', value: 'basic'},
+    {label: 'ECR', value: 'ecr'},
```
Comment (KyleGoyette) [inline docs/guides/launch/prerequisites.md]: This is probably better listed as "AWS" or "EKS"

Comment (KyleGoyette) [reply docs/guides/launch/prerequisites.md]: Since you also configure a Kaniko builder (only used on EKS)

File: docs/guides/launch/prerequisites.md [hunk 4]
```diff
@@ -0,0 +1,148 @@
...
+```
+
+**Environment Flags and Options**
+
+- `type`: specified the environment type, options: `local` and `aws`.
+- `region`: only used with AWS environment, AWS region.
+
+**Registry Flags and Options:**
+
+- `type`: specifies the registry type, options: `local` (default) and `ecr` (ECR requires `aws` environment.
```
Comment (KyleGoyette) [inline docs/guides/launch/prerequisites.md]: Missing closed parantheses here

File: docs/guides/launch/create-job.md [hunk 5]
```diff
@@ -0,0 +1,213 @@
...
+
+There are three ways to create a job:
+
+1. Log a code artifact
+2. Associate a GitHub repository
+3. Set the `WANDB_DOCKER` environment variable
+
+W&B automatically checks your run to see if it can create a job. W&B determines how to create a job based on the following logic:
+
+1. If a GitHub repository is present, create a GitHub sourced job.
```
Comment (KyleGoyette) [inline docs/guides/launch/create-job.md]: This should probably be "a remote Git repo" instead of a "GitHub respository" and it should be a "git" sourced job.

don't want to leave out our friends at Gitlab

File: docs/guides/launch/create-job.md [hunk 6]
```diff
@@ -0,0 +1,213 @@
...
+ ## Create a job from a GitHub repository
+W&B Launch can automatically create a W&B Job if you have a GitHub repository with a Python script that creates a W&B Run. 
+
+:::info
+Unlike the Log a code artifact section [LINK], you do not need to specify `run.log_code()` within your Python script. 
+:::
+
+1. Navigate to your personal or team settings page at [https://wandb.ai/settings](https://wandb.ai/settings) .
+
+2. Select **Connect GitHub**. This will prompt a new screen that confirms you want Weights & Biases to connect to your GitHub account.
```
Comment (KyleGoyette) [inline docs/guides/launch/create-job.md]: Surprisingly, this step is not necessary. We just collect this info for all runs in git repos

File: docs/guides/launch/launch-jobs.md [hunk 7]
```diff
@@ -0,0 +1,248 @@
...
+    {label: 'W&B App', value: 'app'},
+  ]}>
+  <TabItem value="cli">
+Follow the procedure outlined below to recreate a run with the CLI:
+
+1. Find the name of the run you want to reproduce and its associated job name. See the [INSERT] page for information on how to find the job name.
+2. Navigate to your terminal and type the following:
+
+```bash    
+wandb launch -j <job-name> -c path/to/config_file.j
```
Comment (KyleGoyette) [inline docs/guides/launch/launch-jobs.md]: To be consistent we should just say `path/to/config.json`

File: docs/guides/launch/add-jobs-to-queue.md [hunk 8]
```diff
@@ -0,0 +1,81 @@
...
+  values={[
+    {label: 'CLI', value: 'cli'},
+    {label: 'W&B App', value: 'app'},
+  ]}>
+  <TabItem value="cli">
+
+Use the `wandb launch` command to add jobs to a queue. Provide the name of the queue for the `queue`(`-q`) flag, the name of the job for the `job`(`-j`) flag, and the path to the configuration file for the `config`(`-c`) flag.
+
+```bash
+wandb launch -j <job> -q <queue-name> -e <entity-name> -c path/to/config.json
```
Comment (KyleGoyette) [inline docs/guides/launch/add-jobs-to-queue.md:71]: Should we show what an example config would look like here? It should look the same as the modal one.

Comment (KyleGoyette) [reply docs/guides/launch/add-jobs-to-queue.md:71]: Also we may want to warn users, that if they don't add a json file, the script will run with out any args or preset configs passed in.

Comment (KyleGoyette) [reply docs/guides/launch/add-jobs-to-queue.md:71]: We'll eventually add the ability to source from a run in the CLI as well

File: docs/guides/launch/add-jobs-to-queue.md [hunk 9]
```diff
@@ -0,0 +1,81 @@
...
+
+Each run listed corresponds to a job that was was added to that queue. For example, the following image shows there are two jobs listed in a queue called `Starter queue`. One is called `resilient-snowball` and the other is called `earthy-energy-165`:
+
+![](/images/launch/launch_jobs_status.png)
+
+Find additional information about the jobs such as the:
+- **Run**: The name of the W&B Run assigned to that job.
+- **Job ID**: The name of the job. See Job naming conventions[LINK] information on the default naming assigned to a job.
+- **Project**: The name of the project the run belongs to.
+- **Status**: The status of the queue.
```
Comment (KyleGoyette) [inline docs/guides/launch/add-jobs-to-queue.md]: This is the state of the run queue item (or run if that exists). I might just say the status of the queued run?

File: docs/guides/launch/launch-jobs.md [hunk 10]
```diff
@@ -0,0 +1,248 @@
...
+[INSERT]
+
+
+## Recreate a run
+Add your W&B Job to a queue to execute a W&B Run. All jobs pushed to a queue automatically posses the same compute resource type and parameter resource arguments. You can alter the hyperparameters to use when you add a job to a queue.
+
+Interactively recreate a W&B Run with the W&B App or programmatically with the CLI.
+
+<Tabs
+  defaultValue="cli"
```
Comment (KyleGoyette) [inline docs/guides/launch/launch-jobs.md]: I wonder if we should make the default the App here since it is a much easier flow for recreating a run... the CLI doesn't really do that unless you copy in all the config values

File: docs/guides/launch/launch-jobs.md [hunk 11]
```diff
@@ -0,0 +1,248 @@
...
+```    
+This will default to running on the `Docker runner` on the same machine you intend on running on.
+
+:::info
+You need to specify the `run.config` in your config files `overrides.run_config` keys.
+:::
+
+The following code snippet demonstrates an example config file:
+
+```python
```
Comment (KyleGoyette) [inline docs/guides/launch/launch-jobs.md]: should this be JSON?

Also, ignore my previous comment about the sample config file shape

File: docs/guides/launch/launch-jobs.md [hunk 12]
```diff
@@ -0,0 +1,248 @@
...
+    {label: 'Job details', value: 'jobs_details'},
+    {label: 'Version details', value: 'version_details'},
+  ]}>
+  <TabItem value="runs">
+
+Select the name of your job from the list. This will redirect you to a new page with details about each run created by the job such as the:
+
+The Runs tab provides information about each run created by the job such as the:
+
+- **Run**: The names of run.
```
Comment (KyleGoyette) [inline docs/guides/launch/launch-jobs.md]: "The name of the run"

File: docs/guides/launch/launch-jobs.md [hunk 13]
```diff
@@ -0,0 +1,248 @@
...
+- **State**: The state of the run.
+- **Job version**: The version of the job used.
+- **Creator**: Who created the run.
+- **Creation date**: The timestamp of when the run
```
Comment (KyleGoyette) [inline docs/guides/launch/launch-jobs.md]: The timestamp of when the run was started

File: docs/guides/launch/getting-started.md [hunk 14]
```diff
@@ -0,0 +1,162 @@
...
+    print(f"lr: {lr}")
+
+    for epoch in range(2, epochs):
+        # simulating a training run
+        acc = 1 - 2 ** -epoch - random.random() / epoch - offset
+        loss = 2 ** -epoch + random.random() / epoch + offset
+        print(f"epoch={epoch}, acc={acc}, loss={loss}")
+        wandb.log({"acc": acc, "loss": loss})
+
+        run.log_code()
```
Comment (bcsherma) [inline docs/guides/launch/getting-started.md]: Can we move run.log_code() to right after wandb.init()? And put a comment here calling out run.log_code()? Something like 

`# log code so we can reuse this experiment`

File: docs/guides/launch/create-job.md (hunk 6 above)
Comment (ngrayluna) [reply docs/guides/launch/create-job.md]: Mind. Blown.

File: docs/guides/launch/launch-jobs.md (hunk 10 above)
Comment (ngrayluna) [reply docs/guides/launch/launch-jobs.md]: Agreed. :)
